editor/editor_properties_array_dict.cpp

```diff
@@ -412,8 +412,104 @@ void EditorPropertyArray::_remove_pressed(int p_index) {
 	update_property();
 }
 
+void EditorPropertyArray::_button_draw() {
+	if (dropping) {
+		Color color = get_theme_color("accent_color", "Editor");
+		edit->draw_rect(Rect2(Point2(), edit->get_size()), color, false);
+	}
+}
+
+bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
+	String allowed_type = Variant::get_type_name(subtype);
+
+	Dictionary drag_data = p_drag_data;
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "files") {
+
+		Vector<String> files = drag_data["files"];
+
+		for (int i = 0; i < files.size(); i++) {
+			String file = files[i];
+			String ftype = EditorFileSystem::get_singleton()->get_file_type(file);
+
+			for (int i = 0; i < allowed_type.get_slice_count(","); i++) {
```

Comment:
This `i` is shadowing the previous one, which causes a warning.

Reply:
Fixed that up, changed the nested for loop to 'j'